bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/container.jsx

```diff
@@ -18,12 +176,12 @@ export default withTracker(() => {
   } = Service;
 
   return {
+    intlMessages,
```

Comment:
I am not sure why you moved intlMessages into the container. I only see them pulled in the Uploader cocmponent